### DISCUSSION TITLE
'C:\Program' is not recognized as an internal or external command

### DISCUSSION THREAD

**Discussion by FlyingKumquat** [deployphp/deployer#2289] [Help needed]
I'm trying to do a deploy with version 7.0.0-beta.5 like so...
`C:\Dev\project\build>php deployer.phar deploy staging -vvv`

But deployer outputs this...
`[staging] C:\Program Files\PHP\php.exe C:/Dev/project/build/deployer.phar connect --host staging --decorated -vvv`

Which results in the following error to occur...
```
'C:\Program' is not recognized as an internal or external command,
operable program or batch file.
  ⠙ connect staging
In Master.php line 165:

  [Deployer\Exception\ConnectException]
```

Not sure if this is a bug but I do not encounter this issue in version 6.x, is there something I am missing?

**Comment by antonmedv** [comment]
This due to bug in windows exping.

**Comment by antonmedv** [comment]
Maybe you can file a PR to fix it? As for now: use wsl.

**Reply by FlyingKumquat** [reply]
Ended up going with WSL for now :)
